examples/src/main/native/BUILD.bazel

```diff
@@ -18,3 +18,26 @@ cc_binary(
         "@bazel_tools//tools/jdk:jni",
     ],
 )
+
+cc_binary(
+    name = "native_ubsan",
+    srcs = [
+        "com_example_ExampleFuzzerWithNative.cpp",
+        "com_example_ExampleFuzzerWithNative.h",
+    ],
+    copts = [
+        "-fsanitize=fuzzer-no-link,undefined",
+        "-fno-sanitize-recover=all",
+        "-fno-sanitize-coverage=stack-depth",
```

Comment:
@sebastianpoeplau This options really shouldn't be needed. Do you have an idea why the build fails without it?